Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -363,18 +363,35 @@ void JitArm64::WriteExit(u32 destination, bool LK, u32 exit_address_after_return
 
   if (LK)
   {
-    // Push {ARM_PC+20; PPC_PC} on the stack
+    // Push {ARM_PC; PPC_PC} on the stack
     MOVI2R(ARM64Reg::X1, exit_address_after_return);
-    ADR(ARM64Reg::X0, 20);
+    ADR(ARM64Reg::X0, JitArm64BlockCache::BLOCK_LINK_FAST_BL_OFFSET + 12);
     STP(IndexType::Pre, ARM64Reg::X0, ARM64Reg::X1, ARM64Reg::SP, -16);
   }
 
+  constexpr size_t farcode_size = 3 * sizeof(u32);
```

Comment:
Isn't this 3 * sizeof(u32) also the same as JitArm64BlockCache::BLOCK_LINK_SIZE?

Reply:
Yes, but that is a coincidence.